Translations generated in cockpit-machines repo with make update-po

KKoukiou opened the pull request.

marusak commented on machines.html.pot:
```diff
@@ -0,0 +1,11 @@
+"X-Generator: Cockpit html2po\n"
+
+#: src/index.html:4
+msgid "Virtual machines management"
```
Please don't add those partial .pot files

KKoukiou replied:
Done.

marusak reviewed (dismissed):
Please no partial.pot files and also I think this was generated from cockpit repo but it should be generated from https://github.com/cockpit-project/cockpit-weblate to include also languages that are not translated to 50% and thus not in cockpit repo directly.

KKoukiou commented:
> Please no partial.pot files and also I think this was generated from cockpit repo but it should be generated from https://github.com/cockpit-project/cockpit-weblate to include also languages that are not translated to 50% and thus not in cockpit repo directly.

you got it right, did not realize there was this difference!

KKoukiou committed:
Translations copied from cockpit-weblate and regenerated in cockpit-machines repo

This was done with make update-po.

marusak commented:
Looks good, the only issue is that all unused translations are there as well, just commented out. They take 70% of files unnecessary. Can you please remove those? (I guess some mdg* tool will do that with correct option or just you can find the last translated string and delete till the end of file).

KKoukiou committed:
Remove ununused translations

This was automatically done with:

for file in $(ls *.po); do msgattrib --no-obsolete -o $file $file; done

marusak reviewed (approved):
Thanks!
